doc/source/drivers/vector/gtfs.rst

```diff
@@ -39,8 +39,65 @@ Driver capabilities
 
 .. supports_virtualio::
 
+Examples
+--------
+
+List the contents of a GTFS local zip file:
+
+.. tabs::
+
+   .. code-tab:: bash
+
+        ogrinfo GTFS_Realtime.zip
+
+   .. code-tab:: bash gdal CLI
+
+        gdal vector info GTFS_Realtime.zip
+
+List the contents of a GTFS local directory:
+
+.. tabs::
+
+   .. code-tab:: bash
+
+        # specifying the driver
+        ogrinfo GTFS:"/data/GTFS_Realtime"
+        # specifying the input format
+        ogrinfo /data/GTFS_Realtime -if GTFS
+
+   .. code-tab:: bash gdal CLI
+
+        # specifying the driver
+        gdal vector info GTFS:"/data/GTFS_Realtime"
+        # specifying the input format
+        gdal vector info /data/GTFS_Realtime --input-format GTFS
+
+List the contents of a GTFS file at a URL:
+
+.. tabs::
+
+   .. code-tab:: bash
+
+        ogrinfo /vsicurl/https://www.transportforireland.ie/transitData/Data/GTFS_Realtime.zip
+
+   .. code-tab:: bash gdal CLI
+
+        gdal vector info /vsicurl/https://www.transportforireland.ie/transitData/Data/GTFS_Realtime.zip
+
+Extract the stops from the GTFS file and save into a FlatGeobuf file:
+
+.. tabs::
+
+   .. code-tab:: bash
+
+        ogr2ogr -f "FlatGeobuf" stops.fgb GTFS_Realtime.zip stops
```

Comment:
you can omit the explicit output format:
```suggestion
        ogr2ogr stops.fgb GTFS_Realtime.zip stops
```

Reply:
@rouault - would this be the recommended practice for all outputs when we can rely on the filename?

Reply:
yes, the shorter the better (I think)